### ISSUE TITLE
2. The hardest section of code in the project to understand was: (explain what)

### ISSUE THREAD

**Issue by Nicolesc1** [NumEconCopenhagen/projects-2024-victor#2]
I think for question 8, it is a bit hard to understand how the equilibrium is found. More explanation here would have been better.

**Comment by cmv534** [comment]
I also agree that question 8 is hard to understand. There is no description (using #) on the code, making it hard to follow the steps, both on the code and also on economic theory behind it.
